openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/builders/WheelchairGraphStorageBuilder.java

```diff
@@ -201,28 +211,23 @@ private void processSidewalksAttachedToWay(ReaderWay way) {
 	 * way. The same as the attached processing, it looks for the different attributes as tags that are important for
 	 * wheelchair routing and stores them against the generic wheelchair storage object
 	 */
-	private void processWayAsSeparateFeature() {
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.SURFACE);
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.SMOOTHNESS);
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.TRACK);
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.WIDTH);
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.INCLINE);
-
+	private void processWayCheckForSeparateFeature(ReaderWay way) {
+		boolean markSurfaceQualityKnown = isSeparateFootway(way);
+		setWayAttribute(WheelchairAttributes.Attribute.SURFACE, markSurfaceQualityKnown);
+		setWayAttribute(WheelchairAttributes.Attribute.SMOOTHNESS, markSurfaceQualityKnown);
+		setWayAttribute(WheelchairAttributes.Attribute.TRACK, markSurfaceQualityKnown);
+		setWayAttribute(WheelchairAttributes.Attribute.WIDTH, markSurfaceQualityKnown);
+		setWayAttribute(WheelchairAttributes.Attribute.INCLINE, markSurfaceQualityKnown);
 	}
 
 	/**
 	 * Set the specified attribute in the attribute storage object based on the information gathered from the way. This
 	 * method ony sets the attribute in the attribute storage object for the standalone way and not sidewalks.
 	 * @param attribute	The attribute to process
```

Comment:
Should enter the jdoc item here for the markSurfaceQualityKnown attribute

Reply:
done